carma-messenger-ui/website/thirdparty/ros/roslib.js

```diff
@@ -1,3693 +1,6045 @@
-(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
```

Comment:
Trying to get a better understanding of this file and the changes involved, is it automatically generated?

Reply:
It is  a thirdparty ROSLibJS library downloaded from the web.

Reply:
Is there a reason it had to be updated from the old version? Just looking for some description/explanation of the change; I think it would be fine to place this in the PR description.

Reply:
updated